### ISSUE TITLE
ResultEd is missing compare function for sortableTable

### ISSUE THREAD

**Issue by b18jonli** [HGustavs/LenaSYS#7996]
<img width="550" alt="Screenshot_2020-04-08_at_7 33 24_PM" src="https://user-images.githubusercontent.com/62876553/78821028-15461500-79d9-11ea-819b-0a0d9c3de738.png">

**Comment by HGustavs** [comment]
There should be a default compare... either we have one but do not pass it to the callback correctly, or ... the default compare is broken somehow.

**Comment by a18oscte** [comment]
@HGustavs I can't find any compare function other then the one that has been comment-out on line 785. Is that the one?

**Comment by b18jonli** [comment]
I don't think so, the "newCompare" function was (if I understand correctly) created last year to create a generic sorting function for all instances of the sortableTable, but was later changed back to a compare function for each instance. 

It is possible the missing compare function may have been removed and then not changed back.

**Comment by a18oscte** [comment]
@HGustavs am i supposed to write a new compare function?
